client/client-multi/client-c/CMakeLists.txt

```diff
@@ -30,6 +30,14 @@ set(CMAKE_MODULE_PATH
 # 
 # NOTE: Platform specific compilation flags should be set 
 # in the corresponding platform CMake script(s).
+# Sets the default build platform to POSIX.
+if(NOT DEFINED KAA_PLATFORM)
+  set(KAA_PLATFORM "x86-64")
+else()
+  include(${CMAKE_CURRENT_SOURCE_DIR}/toolchains/${KAA_PLATFORM}.cmake)
```

Comment:
CMake toolchain file is included automatically with CMake if CMake cache is empty. You should pass `-DCMAKE_TOOLCHAIN_FILE=/path/to/toollchain/file` to cmake